app/components/Header/index.js

```diff
@@ -127,7 +127,12 @@ class Header extends Component<Props, State> {
     const isLogin = this.state.mode === 'login';
     let title, form;
 
-    if (loggedIn && currentUser && currentUser.selectedTheme !== getTheme()) {
+    if (
+      loggedIn &&
+      currentUser &&
+      ((currentUser.selectedTheme === 'auto' && getTheme() !== getOSTheme()) ||
+        currentUser.selectedTheme !== getTheme())
```

Comment:
If `currentUser.selectedTheme = "auto"`, `getTheme()` returns `light` and `getOSTheme()` returns true;

This would become `loggedIn && currentUser && (currentUser.selectedTheme !== getTheme())` than will essentially be true on each renderer, or am I just overthinking? Not sure how often this component is rendered tho., but might be an issue

Reply:
Oh yea, I could easily fix that